com.quexten.Goldwarden.yml

```diff
@@ -0,0 +1,56 @@
+id: com.quexten.Goldwarden
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: main.py
+finish-args:
+  # Allow network access for sync
+  - --share=network
+
+  # GUI
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+
+  # Lock on screensave
+  - --talk-name=org.gnome.ScreenSaver
+  - --talk-name=org.freedesktop.ScreenSaver
+
+  # Home directory access to setup browser ipc
+  - --filesystem=home
+  # pinentry & approval
+  - --talk-name=org.gnome.keyring.SystemPrompter
+  # biometric / user password auth
+  - --system-talk-name=org.freedesktop.PolicyKit1
+modules:
+  - ./python3-requirements.json
+  - name: wl-clipboard
+    buildsystem: simple
```

Comment:
this should be `meson` for the build system

Reply:
Thanks, switched to meson now!